index.js

```diff
@@ -32,15 +32,49 @@ PLAYERS.forEach(renderPlayer)
 
 
 /***** Deliverable 1 *****/
+const header = document.querySelector("h1#header")
 function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+  if (element.target.style.color === "red") {
+    element.target.style.color = "black"
   } else {
-    element.style.color = "red"
+    element.target.style.color = "red"
   }
 }
```

Comment:
Nice work on this callback! Here, you'd want to use a different name for the parameter instead of element to make it clearer what's being passed into the function:

```js
function toggleColor(event) {
  if (event.target.style.color === "red") {
    event.target.style.color = "black"
  } else {
    event.target.style.color = "red"
  }
}
```